tools/documentation_crawler/documentation_crawler/spiders/check_help_documentation.py

```diff
@@ -34,11 +34,25 @@ def _is_external_url(self, url: str) -> bool:
         return is_external or self._has_extension(url)
 
     def closed(self, *args: Any, **kwargs: Any) -> None:
-        unused_images = set(os.listdir(self.help_images_static_dir)) - self.static_images
+        unused_images = set(os.listdir(self.images_static_dir)) - self.static_images
         if unused_images:
-            exception_message = "The following images are not used in help documentation " \
+            exception_message = "The following images are not used in documentation " \
                                 "and can be removed: {}"
             self._set_error_state()
             unused_images_relatedpath = [
-                os.path.join(self.help_images_path, img) for img in unused_images]
+                os.path.join(self.images_path, img) for img in unused_images]
             raise Exception(exception_message.format(', '.join(unused_images_relatedpath)))
+
+
+class APIDocumentationSpider(HelpDocumentationSpider):
```

Comment:
We should make both of these subclasses of a common class, rather than subclassing like this.